### DISCUSSION TITLE
Github pages deployment doesn't start

### DISCUSSION THREAD

**Discussion by JungsGames** [community/community#21991] [Other Feature Feedback, Questions, & Ideas]
<p>Hello Team,</p>
<p>My public GitHub pages did not get deployed after I committed changes on two different occations.</p>
<p>Can anyone help me check this issue?</p>

**Comment by jfullstackdev** [comment]
<p>where is that repo ? private ?</p>
<p>because you only have this</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/b/ab1227a68895c6a7158b104104306b588882f47d.png" data-download-href="/uploads/short-url/opmxJydqUanYdlkPus9la3vQl0V.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181068083-c1e6ecfe-c700-49c3-b0f0-44a78a8e2d06.png" alt="image" data-base62-sha1="opmxJydqUanYdlkPus9la3vQl0V" width="562" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/b/ab1227a68895c6a7158b104104306b588882f47d_2_562x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/b/ab1227a68895c6a7158b104104306b588882f47d_2_843x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/b/ab1227a68895c6a7158b104104306b588882f47d_2_1124x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/b/ab1227a68895c6a7158b104104306b588882f47d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1170×1040 118 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>last update was 2016</p>

**Comment by JungsGames** [comment, marked as answer]
<p>Hello,</p>
<p>This issue is resolved.</p>
<p>It didn’t deploy because I’m using a private repo from a free account. Github pages require a premium account to deploy from a private repository.</p>
